server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -932,3 +928,27 @@ timestamp_precision=ms
 ####################
 # Datatype: float
 # group_by_fill_cache_size_in_mb=1.0
+
+####################
+### Schema Engine Configuration
+####################
+# Choose the mode of schema engine. The value could be Memory and Schema_File. If the provided value doesn't match any pre-defined value, Memory mode will be used as default.
+# Datatype: string
+# schema_engine_mode=Memory
+
+####################
+### Schema File Configuration
+####################
+# the max num of thread used for flushing metadata to schema file
+# Datatype: int
+# max_schema_flush_thread=15
+
+# The minimum size (in bytes) allocated for a node in schema file
+# A large number for this will make it faster while occupying more space, or otherwise
+# Datatype: short
+# minimum_schema_file_segment_in_bytes=0
```

Comment:
It means if a flushed internal(entity) node has less than 20 child, it will get a segment with the size calculated from total size of its children. 
If it has no child, it will get a segment of 25 bytes, which is the size of segment header.